lighthouse-core/lib/network-request.js

```diff
@@ -9,6 +9,31 @@
  * @fileoverview Fills most of the role of NetworkManager and NetworkRequest classes from DevTools.
  * @see https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/sdk/NetworkRequest.js
  * @see https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/sdk/NetworkManager.js
+
+  DevTools box-whisker
+
+    |-------[xxxxxXXXXXX]-|
+       (1)    (2)    (3) (4)
+
+  (1) Queuing (delta between renderer knowing about request and network manager knowing about it),
```

Comment:
nit: seems to follow the pattern here
```suggestion
  (1) Leading whisker: Queuing (delta between renderer knowing about request and network manager knowing about it),
```